Snakefile

```diff
@@ -15,6 +15,7 @@ if not exists("config.yaml"):
     copyfile("config.default.yaml", "config.yaml")
 
 
+configfile: "config.pypsa-earth.yaml"
```

Comment:
Yes, basically it loads each config file and for any new config, the changed parameters are overwritten.
This may allow the config.yaml to be even  lighter and keep only the needed changes.
The goal of this is to have all pypsa-earth parameters within the simulation and will also help the merge as there is no need for duplication.

In some scripts, some pypsa-earth parameters were needed but that required the manual adjustment of the config.pypsa-earth.yaml and config.yaml independently.
With this is no more needed: the parameter shall stay in config.pypsa-earth
For context, in pypsa-eur they now moved to a similar setup.

In the past, we considered this solution in pypsa-earth but the "merge" of configs was not smooth as it is now